### PR TITLE
output TaskLogs to jobid folder

### PR DESCRIPTION
**Pull request by marcocast**

### CODE REVIEW

**Comment by marcocast** [comment]
retest this please

**Comment by paraita** [comment]
squash the 2 commits (they have the same comment) everything else is fine

File: rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java [hunk 1]
```diff
@@ -1299,8 +1301,8 @@ public InputStream jobFullLogs(@HeaderParam("sessionid") String sessionId,
 
     public InputStream retrieveTaskLogsUsingDataspaces(String sessionId, String jobId, TaskId taskId)
```
Comment (sonarqube-activeeon) [inline rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java]: MAJOR Remove this unused method parameter "jobId". [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1172)

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 1 issue
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Pull request merged**